Sources/SwiftDocC/DocumentationService/Convert/ConvertService.swift

```diff
@@ -150,6 +150,18 @@ public struct ConvertService: DocumentationService {
             context.allowsRegisteringArticlesWithoutTechnologyRoot = true
             context.allowsRegisteringUncuratedTutorials = true
             
+            if let overridingDocumentationComment = request.overridingDocumentationComment {
+                context.configureSymbolGraph = { symbolGraph in
+                    for (identifier, symbol) in symbolGraph.symbols {
+                        var symbol = symbol
+                        symbol.docComment = SymbolGraph.LineList(
+                            overridingDocumentationComment.map(SymbolGraph.LineList.Line.init(_:))
+                        )
+                        symbolGraph.symbols[identifier] = symbol
+                    }
```

Comment:
I think this creates temporary copies of each symbol that it iterates over that could be avoided by updating the symbol via subscripting.

This probably doesn't matter to much in practice since the convert service symbol graphs are typically very small.

Also, you can create the LineList for the overridingDocumentationComment once since it's the same for all symbols.

```suggestion
                    let newDocComment = SymbolGraph.LineList(
                         overridingDocumentationComment.map(SymbolGraph.LineList.Line.init(_:))
                    )
                    for identifier in symbolGraph.symbols.keys {
                        symbolGraph.symbols[identifier].docComment = newDocComment
                    }
```

Reply:
That's a great point, thanks!